src/components/Hero/ProfileChart/RectChart/index.js

```diff
@@ -109,20 +109,20 @@ function ReactChart({
 }
 
 ReactChart.propTypes = {
-  delayed: PropTypes.number,
+  behind: PropTypes.number,
```

Comment:
Shouldn't these also change to `behindSchedule`?

Reply:
@kilemensi Yap changed them